app/javascript/dashboard/components/layout/Sidebar.vue

```diff
@@ -104,6 +104,9 @@ export default {
       return '';
     },
     customViews() {
+      if (!this.activeCustomView) {
```

Comment:
Added this as it was breaking the Sidebar (old) when moving from Conversations to Settings pages.